source/common/access_log/access_log_impl.h

```diff
@@ -229,6 +229,24 @@ class GrpcStatusFilter : public Filter {
   protoToGrpcStatus(envoy::config::accesslog::v3::GrpcStatusFilter::Status status) const;
 };
 
+/**
+ * Filters requests based on access log type
+ */
+class LogTypeFilter : public Filter {
+public:
+  using LogTypeHashSet = absl::node_hash_set<AccessLogType, absl::Hash<AccessLogType>>;
```

Comment:
Use flat_hash_map?

Reply:
Did you mean flat_hash_set? Used flat_hash_set for now, please lmk if I misunderstood @lizan.